spectacles/lookml.py

```diff
@@ -410,7 +440,7 @@ def build_project(
     fields = ["name", "project_name", "explores"]
     for lookmlmodel in client.get_lookml_models(fields=fields):
```

Comment:
I'm not sure how important this is, but we loop through models 4 times (441, 459, 467, 477). Do you think we can consolidate some of these loops? I don't consider it blocking for the PR but if you have some time it might give us a bit of efficiency and readability.